test/tif-reproject.test.js

```diff
@@ -63,3 +64,11 @@ test('[tif-reproject] reprojection: to epsg:3857', function(assert) {
     assert.end();
   });
 });
+
+test('[tif-reproject] reprojection: failed due to unsupported srs', function(assert) {
+  reproject(unsupported, { filetype: 'tif' }, function(err) {
+    assert.ok(err, 'expected error');
+    assert.equal(err.message, 'Unsupported SRS', 'expected error message');
```

Comment:
I believe these messages end up in the user's 👀 . Should was suggest the user reproject to epsg:3857?